backend/src/main/java/ro/utcn/sd/prof/controller/AppUserController.java

```diff
@@ -0,0 +1,41 @@
+package ro.utcn.sd.prof.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import ro.utcn.sd.prof.command.Cmd;
+import ro.utcn.sd.prof.dto.AppUserDTO;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class AppUserController {
+
+    private final Cmd cmd;
+
+    @GetMapping("/me")
+    public AppUserDTO readCurrent() {
+        return (AppUserDTO)cmd.execute("viewDetails", null).get(0);
+    }
+
+    @PostMapping("/users")
+    public AppUserDTO createUser(@RequestBody AppUserDTO dto){
+        return (AppUserDTO)cmd.execute("createUser", dto).get(0);
+    }
+
+    @DeleteMapping("/users")
```

Comment:
Usually, delete methods have a URL parameter for the ID and no request body (`/users/{id}`).